java-components/resource-model/src/main/java/com/redhat/hacbs/resources/model/v1alpha1/BuildRecipe.java

```diff
@@ -10,6 +10,8 @@ public class BuildRecipe {
 
     private String enforceVersion;
 
+    private List<String> ignoredArtifacts;
```

Comment:
I should make a note to update the java side of BuildRecipe when I finish https://github.com/redhat-appstudio/jvm-build-service/pull/118